headers/types/common/common.h

```diff
@@ -146,80 +139,6 @@ struct rgb {
 };
 ASSERT_SIZE(struct rgb, 4);
```

Comment:
> I think rect16_xywh and rgb are quite generic structs that could be used outside of the context of file parsing. Maybe there should be a file for generic structures (but that probably should already what common is about...)

I would still like to move these to `file_io.h` because they're not actually used in a generic fashion. As far as I know, they're only used for reading WTE files, so it makes sense for them to be defined in the same place as all the wte structs. If we find out they're used elsewhere, we can move them back later.